pilz_control/test/unittest_pilz_joint_trajectory_controller.cpp

```diff
@@ -75,33 +151,160 @@ void PilzJointTrajectoryControllerTest::SetUp()
  *    1. Construct the controller and call init().
  *
  * Expected Results:
- *    1. The force hold service exists and the controller is in mode HOLDING.
+ *    1. The hold/unhold services and the is_executing service exist.
```

Comment:
please call it either `is_executing` or `query_activity` consistently 
(I would prefer the first one)